specification/search/resource-manager/Microsoft.Search/preview/2024-03-01-preview/search.json

```diff
@@ -2604,76 +2604,6 @@
       },
       "description": "The details of a long running asynchronous shared private link resource operation"
     },
-    "ErrorResponse": {
-      "type": "object",
-      "properties": {
-        "error": {
-          "$ref": "#/definitions/ErrorDetail",
-          "description": "Describes a particular API error with an error code and a message."
-        },
-        "message": {
```

Comment:
The ErrorResponse defined here is different than the error defined in common, I don't think we should take this change.

Reply:
ack. Reverting. We can keep this in mind for the future